eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -183,6 +183,8 @@ jobs:
     variables:
       - template: ../variables/globals.yml
 
+    condition: and(succeeded(), ne(variables['Skip.Test'], 'true'))
```

Comment:
This looks like an errand copy/paste.

Reply:
Wasn't errant. It's not a negative (this isn't set to true by default) but I needed to get through the failing unit tests.

Reply:
I couldn't tell where this applies and now see it is attached to the Job it just looks odd floating out here.